db/migrate/20151028125334_create_supervisions.rb

```diff
@@ -7,6 +7,6 @@ def change
       t.timestamps null: false
     end
 
-    add_index :supervisions, [:mission_id, :supervisor_id], unique: true
+    add_index :supervisions, [:mission_id, :supervisor_id], where: "accepted = 1", unique: true
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.